pumps/kafka.go

```diff
@@ -137,9 +142,21 @@ func (k *KafkaPump) Init(config interface{}) error {
 		k.log.WithField("SASL-Mechanism", k.kafkaConf.SASLMechanism).Warn("Tyk pump doesn't support this SASL mechanism.")
 	}
 
+	// Timeout is an interface type to allow both time.Duration and float values
```

Comment:
should we add a default value here? there wasn't a default one before this change